compiler/plugins/target/LLVMCPU/LLVMCPUTarget.cpp

```diff
@@ -467,6 +477,8 @@ class LLVMCPUTargetBackend final : public TargetBackend {
 
     // Dump just the codegen bitcode before linking and optimization.
     if (!options.dumpIntermediatesPath.empty()) {
+      dumpLLVMIRToPath(options.dumpIntermediatesPath, options.dumpBaseName,
+                       variantOp.getName(), ".codegen.ll", *llvmModule);
       dumpBitcodeToPath(options.dumpIntermediatesPath, options.dumpBaseName,
                         variantOp.getName(), ".codegen.bc", *llvmModule);
```

Comment:
We could update the docs here after this change: https://iree.dev/developers/general/developer-tips/#__tabbed_1_1

* include `.ll` files in sample output
* clarify that `llvm-dis` on the `.bc` files isn't needed when the `.ll` files already exist

![image](https://github.com/iree-org/iree/assets/4010439/517ff4d9-625b-4c20-8139-298ea0773f1e)

Source for that is at https://github.com/iree-org/iree/blob/main/docs/website/docs/developers/general/developer-tips.md